src/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  width: 96px;
```

Comment:
Don't hardcode block width, use margin-right: 4px between stars instead (also no need to add margin-right to the last stars in a row)